styx-common/src/main/java/com/spotify/styx/storage/Storage.java

```diff
@@ -79,6 +79,11 @@
   boolean debugEnabled() throws IOException;
 
   /**
+   * Get the submission rate for Styx.
```

Comment:
Not completely informative. It is "maximum rate" or something, not simply "rate". The documentation should describe units, too.